### PR TITLE
Grammar

### PR DESCRIPTION
**Pull request by belottir**
I propose to replace greating with greatly, which seems more appropriate to me.

### CODE REVIEW

File: dev/source/docs/building-setup-windows10.rst [hunk 1]
```diff
@@ -88,5 +88,5 @@ You should now be able to start the "Ubuntu" application from your Windows Start
    
 .. tip::
 
-    Windows 10 includes "Windows Defender" virus scanner. It will slow down WSL quite a bit. Disabling greating improves disk performance but increases your risk to viruses so disable at your own risk. Here is one of many resources/videos that show you how to disable it: https://www.youtube.com/watch?v=FmjblGay3AM
+    Windows 10 includes "Windows Defender" virus scanner. It will slow down WSL quite a bit. Disabling greatly improves disk performance but increases your risk to viruses so disable at your own risk. Here is one of many resources/videos that show you how to disable it: https://www.youtube.com/watch?v=FmjblGay3AM
```
Comment (Hwurzburg) [inline dev/source/docs/building-setup-windows10.rst]: thanks....why dont we replace "greating" with "it greatly" for a clearer note on what exactly is to be disabled...will be happy to merge it then!

Comment (belottir) [reply dev/source/docs/building-setup-windows10.rst]: I'm sorry @Hwurzburg but I'm new to git/github. Should I make another patch?

**Comment by Hwurzburg** [comment]
dont worry about it...I will merge and fix it tomorrow....thanks

**Comment by Hwurzburg** [comment]
merged...thanks!

**Pull request merged**